tests/selenium/spec/signin-spec.js

```diff
@@ -0,0 +1,54 @@
+const SignInPage = require("../framework/page-objects/SignInPage");
+const util = require('../framework/shared/util');
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+describe('Sign in page check spec', () => {
+  const signInPage = new SignInPage();
+
+  beforeEach(util.itHelper(async () => {
+    await signInPage.navigate();
+  }));
+
+  it('shows all the buttons', util.itHelper(async () => {
+    expect(await signInPage.isSignInWithEmailButtonPresent(), 'expects login with email button to be present').to.equal(true);
```

Comment:
we can also extend the test to check if the text of the button is present in dom ( maybe for next time )